docs/src/static-pages/accessibility.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Developer Accessibility Guidelines
+subtitle: To ensure that we are building the best possible products that work for everyone, we need to consider accessibility at every stage.
+category: guidelines
+---
+
+# Accessible mixins for Web
+
+Within Backpack there are a number of SASS mixins available for use to enable content to be more accessible.
+
+[`bpk-hidden`](https://backpack.github.io/sassdoc/#mixin-bpk-hidden)
+
+Used to hide visually and from screen readers.
+
+[`bpk-visually-hidden`](https://backpack.github.io/sassdoc/#mixin-bpk-visually-hidden)
+
+Hide only visually, but have it available for screen readers: http://snook.ca/archives/html_and_css/hiding-content-for-accessibility
```

Comment:
```suggestion
Hide only visually, but have it available for screen readers. [Learn more &raquo;](http://snook.ca/archives/html_and_css/hiding-content-for-accessibility) 
```

Let's use a nicely-formatted link here instead of just putting the raw URL.